src/core/thread/network_diagnostic.cpp

```diff
@@ -430,7 +430,7 @@ Error NetworkDiagnostic::FillRequestedTlvs(const Message &       aRequest,
 #if OPENTHREAD_FTD
         case NetworkDiagnosticTlv::kMaxChildTimeout:
         {
-            uint32_t maxTimeout;
+            uint32_t maxTimeout = 0;
```

Comment:
Should we update `GetMaxChildTimeout()` so that it does not require the client to initialize?

Reply:
+1

Reply:
Updated.